Dockerfile

```diff
@@ -9,9 +9,17 @@ COPY . /src/blockstack
 WORKDIR /src/blockstack
 
 # Install Dependancies
-RUN apt-get update && apt-get install -y python-pip python-dev build-essential apt-utils libssl-dev libffi-dev rng-tools libgmp3-dev sudo software-properties-common sqlite3 lsof
+RUN apt-get update && apt-get install -y python-pip python-dev build-essential apt-utils libssl-dev libffi-dev rng-tools libgmp3-dev sudo software-properties-common sqlite3 lsof curl wget
+
+# Lets add blockstack-browser to the mix
+RUN wget -qO - https://raw.githubusercontent.com/blockstack/packaging/master/repo-key.pub | apt-key add -
```

Comment:
If we are going to install the browser from our `apt` repo I think we might want to modify the `blockstack-core` install here to use the `apt` repo too.